content/news/buzz/2017-google-summer-of-code-final-submission-adam-boniecki/contents.lr

```diff
@@ -0,0 +1,54 @@
+title: 2017 Google Summer of Code - Final Submission
+---
+author: Adam Boniecki
+---
+body:
+
+With Google Summer of Code 2017 program nearing its end, it is time to summarize what I got done during the summer working on `Batavia </project/projects/bridges/batavia/>`__.
+
+Batavia is a part of Pybee's BeeWare collection of projects. I posted my proposal in `this GitHub thread <https://github.com/pybee/batavia/issues/483>`__ and kept it updated with my progress on a weekly basis.
```

Comment:
Even though people could click the link and read the proposal, it would be awesome to have a sentence or two of high-level summary of what you _wanted_ to accomplish when you set out this summer.